Automattic/simplenote-android
# Tweak screenshots automation to shot note with links instead of inter-note linking view

### Fix
@SylvesterWilmott pointed out that, because we don't have descriptions in the screenshots in the Play Console, the shot with the inter-note linking view might seem confusing to potential users.

<img width="300" alt="Screen Shot 2021-03-31 at 4 14 43 pm" src="https://user-images.githubusercontent.com/1218433/113093862-7004fc00-923c-11eb-8f62-4ffbf12a89c6.png">

We decided to use a screenshot of a note with internal links instead (p2XJRt-2t6-p2#comment-4784):

<img alt="image" src="https://user-images.githubusercontent.com/1218433/113093907-83b06280-923c-11eb-8656-38c99100e28d.png" width=300/>

### Test

- Checkout this branch
- Run `bundle install`
- Launch a Simulator
- Run `bundle exec fastlane take_screenshots`

The process should eventually open your browser and show something like this:

![image](https://user-images.githubusercontent.com/1218433/113094043-bc503c00-923c-11eb-9733-46dba701f6a7.png)

Note that:

- It's not 100% stable, so if it fails try giving it another go. _This is absolutely awful, but let's be pragmatic and keep in mind that_ at the moment _this automation is used only for the occasional screenshot generation, so investing too much time in making it bulletproof is,_ unfortunately, _not on top of our priority list_.
- At least once, I got the new screenshot _without_ the syntax highlight for the links. Not sure how that happened. Again, I re-run the tests and it came out fine.

### Review
Only one developer required to review these changes, but anyone can perform the review.


### Release
These changes do not require release notes.
